src/components/Summary/QuestionComment/index.js

```diff
@@ -10,54 +13,51 @@ class QuestionComment extends React.Component {
         <h4 style={{ display: 'inline', color: 'red' }}>Not Available !</h4>
       );
     }
-
     return (
       <>
-        <h3>Interviewer：{interviewer}</h3>
+        <h3 style={{ fontSize: 20 }}>Interviewer：{interviewer}</h3>
```

Comment:
`fontSize` should have its unit.
And if you're using `h3`, which is a native HTML element. Keep its size as it will be better.